_posts/2023-10-28-optimize-refresh-interval.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  "Optimize OpenSearch Refresh Interval"
+authors:
+  - ev2900
+  - awssamit
+date: 2023-10-28
+categories:
+  - technical-posts
+meta_keywords: OpenSearch index, OpenSearch refresh, refresh interval optimization, optimize OpenSearch index performance

```

Comment:
Please update the meta with the following: 

**meta_keywords:** OpenSearch refresh interval, refresh interval optimization, optimie OpenSearch index performance

**meta_description:** Learn how to optimize the refresh interval of an OpenSearch index and strike a balance between the speed at indexed information is available for search with CPU and I/O costs.

Reply:
I made these changes